.github/workflows/ci.yml

```diff
@@ -285,13 +310,11 @@ jobs:
           branch: ${{ inputs.branch }}
           deploy_id: ${{ github.run_number }}
           deploy_root: app/${{ inputs.deploy_root }}
-          deploy_labels: gh/workflow-version=${{ needs.build.outputs.workflow_ci_version }} gh/workflow-run=${{ github.run_id }}-${{ github.run_number }}-${{ github.run_attempt }}
+          deploy_labels: gh/workflow-version=${{ needs.build.outputs.workflow_ci_version }}
```

Comment:
1. Can you explain what was being done in simple english before (and potentially why - see if the commit history can reveal or how its being used in existing repo)?
2. Can you explain what is being done in simply english and why is this change needed with this PR (aka with adding support for AWS)?

Reply:
To be clear, my question was around changing `deploy_labels`.

Can you try to answer my questions from that point of view?